development/nginx/command

```diff
@@ -1 +1,7 @@
+data='{"alert": false, "summary": "convert2rhel did not detect issues", "report": "", "report_json": {"foo": "bar"}}'
+
 /usr/bin/convert2rhel --help
+echo "BEGIN MARKER"
+echo "$data"
+echo "END MARKER"
+/usr/bin/convert2rhel --version
```

Comment:
I would only invert the order of execution here... Ideally, the marker will be the last thing in the output, and the tool that is running will come just before it. 

Thinking about the idea that the script will "read" the output of the tool that just run, I think it makes sense that the tool runs before the marker. Let me know what you think.

Reply:
Sure, I have no problem with that, when I was reading the description I just though that if it should be generic that It shouldn't matter to insights where the output is in the stdout as long as it is between the markers it will get shown. That is why I put command before and after the markers.

Reply:
Inverted, commands are firsts then the content within markers